third_party/ibis/ibis_oracle/datatypes.py

```diff
@@ -47,6 +48,14 @@ class _FieldDescription(TypedDict):
     null_ok: Optional[int]
 
 
+class ORACLE_BOOLEAN(sqltypes.TypeEngine):
```

Comment:
Since dvt relies on SQLAlchemy, we had to create a custom type because BOOLEAN is not supported. I wonder if similar issues could occur with other databases as well. Would it make sense to move this into the core and define it as a shared type for unsupported common types?

Reply:
SQLAlchemy does already define a BOOLEAN and some other systems already use it, for example PostgreSQL.

I could have suggested we import `sqltypes.BOOLEAN` instead and we could have used that further down this file. I decided to suggest a new class but, in hindsight, I think your suggestion is better.

I suggest you remove the `ORACLE_BOOLEAN` class and change the new `dt.dtype.register` call to use `sqltypes.BOOLEAN`.

Reply:
@nj1973 Thanks for guidance, I'll make it soon.